Makefile

```diff
@@ -40,121 +40,88 @@ ifndef SR_XML
 	SR_XML=XE_SR_ERRORCODES.xml
 endif
 
-# OASIS_START
+build:
+	jbuilder build c/gen_c_binding.exe
```

Comment:
Can this be done in one call? Not sure it is supported but it should offer better performance.

```
ALL += c/gen_c_binding.exe
ALL += csharp/gen_csharp_binding.exe
..
build:
  jbuilder build $(ALL)
```